optuna/integration/tensorboard.py

```diff
@@ -49,8 +49,19 @@ def __call__(self, study: optuna.study.Study, trial: optuna.trial.FrozenTrial) -
         run_name = "trial-%d" % trial.number
         run_dir = os.path.join(self._dirname, run_name)
         with tf.summary.create_file_writer(run_dir).as_default():
-            hp.hparams(hparams)  # record the values used in this trial
-            tf.summary.scalar(self._metric_name, trial_value, step=1)
+            hp.hparams_config(
+                hparams=list(hparams.keys()),
+                # Control metric values displayed on HPARAMS tab of TensorBoard.
+                metrics=[
+                    hp.Metric(
+                        tag=self._metric_name,
+                        display_name=self._metric_name,
+                        description="Trial value optimized with Optuna.",
```

Comment:
Though `description` is an interesting feature, it looks redundant just to show the same message. How about removing `description` so far?

Reply:
OK. Which do you think better?

- Remove `hp.hparams_config`
- Remove only `description` option

Reply:
In my understanding, users will get the same output with or without `hp.params_config` if we remove `description` option. If so, I prefer shorter code.

Reply:
I also prefer shorter code.

@sfujiwara 
Would you mind removing `hp.hparams_config`?

Reply:
No problem. I removed `hp.hparams_config`!
Removing `hp.hparams_config` does not change the result.
You can see the result is here: https://tensorboard.dev/experiment/MlImIJjdQaOX0gEM774BEA/